src/identity/components/OrganizationListTab/OrganizationCard.scss

```diff
@@ -7,6 +7,16 @@
   border-radius: 2px;
 }
 
+.account--organizations-tab-suspended-orgs-card {
+  margin-top: $cf-space-3xs;
+  margin-bottom: $cf-space-3xs;
+  border-radius: 2px;
+  background-color: #0e0e0e;
+  div:not(:last-child) {
+    opacity: 0.5;
+  }
+}
```

Comment:
If you change the `<a>` tag below to a `SafeBlankLink`, would you mind coming in here and alphabetizing the properties here please?